frontend/src/components/util/resetComponent.tsx

```diff
@@ -9,34 +9,27 @@ import { StatusCodes } from "http-status-codes";
 import { AxiosResponse } from "axios";
 import timers from "../../properties/timers";
 import { useEditionApplicationPathTransformer } from "../../hooks/utilHooks";
-import { useCurrentUser } from "../../hooks/useCurrentUser";
-import { emptyUser } from "../../api/entities/UserEntity";
+import { IUser } from "../../api/entities/UserEntity";
 
 interface ResetComponentProps {
     name: string;
-    handler: (url: string, email: string) => Promise<AxiosResponse>;
+    handler: (url: string, newValue: string) => Promise<AxiosResponse>;
+    user: IUser;
+    token: string | null;
```

Comment:
You probably want:
```suggestion
    token?: string;
```
 This makes the argument non-mandatory.